pkg/activate/activator.go

```diff
@@ -145,3 +148,26 @@ func (b *ActivatorImpl) applyActivatorStrategy(p *kapiv1.Pod) error {
 	}
 	return nil
 }
+
+//GetStatus return the status for the breaker
+func (b *ActivatorImpl) GetStatus() v1.BreakerStatus {
```

Comment:
Link to the previous comment, the signature of this method should be: 
`func (b *ActivatorImpl) GetStatus() v1.ActivatorStatus`